src/loaders/webp/meson.build

```diff
@@ -1,5 +1,14 @@
 webp_deb = []
 
+config_h = configuration_data()
+
+config_h.set('WEBP_FORCE_ALIGNED', true)
```

Comment:
Instead of making the script complex, how about we enable it directly in the header (bit reader). The point is this script is not a configurable at all.

Reply:
Hi I updated this PR.
The google lib is no longer use unaligned memory access. so i found 2 commit related this. and i applied it.